README.md

```diff
@@ -1 +1,46 @@
-# silverstripe-tx-translator
+# silverstripe tx translator
+
+Module to assist with transifex integration. Works with Silverstripe 4.12+ and 5.0+.
+
+## Operations
+
+- Pull latest translations from transifex and merge them into yml/json/js translation files
+- Run i18nTextCollectorTask on a local website
+- Push updated source strings to transifex (optional)
+- Create github pull-requests with file changes
+
+## Requirements
+
+Transifex cli client (tx) - [https://developers.transifex.com/docs/cli](https://developers.transifex.com/docs/cli) configured API key from transifex.
+
+You must use the go version at least version 1.6+ and not the old python version.
+
+Create a new classic github api token in [github token settings](https://github.com/settings/tokens) with all checkboxes unticked except for `public_repo`. This is required to create pull-requests.
+
+Delete this token from github once you have completed updating translations.
+
+## Usage
+
+```
+cd /path/to/my-local-site
+composer install silverstripe/tx-translator
+TX_GITHUB_API_TOKEN=mytoken TX_SITE=my-local-site.test TX_PUSH=1 php vendor/silverstripe/tx-translator/scripts/translate.php
+```
+
+**To omit the push to transifex**
+
+```
+TX_GITHUB_API_TOKEN=mytoken TX_SITE=my-local-site.test php vendor/silverstripe/tx-translator/scripts/translate.php
+```
+
+## Environment variables
+
+Environment variables can either be set via the command line, like in the examples above, or in an .env in the root folder of the site, or using any other methods to set environment variables.
+
+Note that the only valid values for the boolean variables is 1, true, 0, and false.
```

Comment:
```suggestion
Note that the only valid values for the boolean variables are "1", "on", "true", "0", "off", and "false".
```
Sorry, should have added quotes the first time around - but I'm only adding them now because a change is needed anyway to add "on" and "off"

Reply:
You don't actually need to quote them in the .env file, and you definitely don't when passing on cli, so we're probably better off leaving them unquoted